src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/WorkerTest.java

```diff
@@ -435,6 +435,8 @@ public final void testInitializationFailureWithRetries() throws Exception {
         String stageName = "testInitializationWorker";
         IRecordProcessorFactory recordProcessorFactory = new TestStreamletFactory(null, null);
         config = new KinesisClientLibConfiguration(stageName, null, null, WORKER_ID);
+        config.withMaxInitializationAttempts(2);
```

Comment:
In setup(), we initialize the global config, which is used by most of the test cases. Shall we also set the max attempts for this global config?

Reply:
Sure, I'll set a global config to 1 and override it where appropriate (only this unit test currently)